src/MAUI/Maui.Samples/Samples/UtilityNetwork/CreateLoadReport/CreateLoadReport.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage x:Class="ArcGIS.Samples.CreateLoadReport.CreateLoadReport"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui"
+             xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.Maui">
+    <Border Style="{DynamicResource EsriSampleControlPanel}">
+        <VerticalStackLayout Spacing="5">
+            <Label Text="Add phases from the dropdown list, then run report." />
+            <Picker x:Name="PhasesList" ItemDisplayBinding="{Binding Name}" />
+            <Button Clicked="OnAddPhase" Text="Add" />
+            <Label x:Name="Phases"
+                   BackgroundColor="{AppThemeBinding Dark=Gray}"
+                   HorizontalTextAlignment="Center" />
+            <Button Clicked="RunReportButton_Clicked" Text="Run Report" />
+            <Button Clicked="OnReset" Text="Reset" />
+            <CollectionView x:Name="ReportView"
+                            Margin="0,10,0,0"
+                            BackgroundColor="{AppThemeBinding Dark=Gray}">
```

Comment:
The report will be cut off in horizontal mode on mobile if the user selects enough phases. A vertical scrollbar and a height request remedy this. For more space for report, consider condensing UI controls.